Debounce/BufferedEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// SPDX-FileCopyrightText: 2024 Frans van Dorsselaer
+//
+// SPDX-License-Identifier: MIT
+//
+// SPDX-FileContributor: Alain van den Berg
+
+namespace Dorssel.Utilities;
+
+/// <summary>
+/// Provides data for the <see cref="IBufferer{TEvent}.Buffered"/> event.
+/// </summary>
+public class BufferedEventArgs<TEvent> : EventArgs
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="BufferedEventArgs{TEvent}"/> class.
+    /// </summary>
+    /// <param name="events">The original events accumulated since the previous buffered event was sent.
+    /// <para>Must be greater than 0.</para>
+    /// </param>
+    public BufferedEventArgs(IList<TEvent> events)
```

Comment:
I think an `IReadOnlyList` is better here. It gets passed on (hopefully unmodified) to possibly multiple listeners.

Reply:
You're right, fixed.